modules/setting/oauth2.go

```diff
@@ -90,23 +90,25 @@ func parseScopes(sec ConfigSection, name string) []string {
 }
 
 var OAuth2 = struct {
-	Enabled                    bool
-	AccessTokenExpirationTime  int64
-	RefreshTokenExpirationTime int64
-	InvalidateRefreshTokens    bool
-	JWTSigningAlgorithm        string `ini:"JWT_SIGNING_ALGORITHM"`
-	JWTSigningPrivateKeyFile   string `ini:"JWT_SIGNING_PRIVATE_KEY_FILE"`
-	MaxTokenLength             int
-	DefaultApplications        []string
+	Enabled                     bool
+	AccessTokenExpirationTime   int64
+	RefreshTokenExpirationTime  int64
+	InvalidateRefreshTokens     bool
+	JWTSigningAlgorithm         string `ini:"JWT_SIGNING_ALGORITHM"`
+	JWTSigningPrivateKeyFile    string `ini:"JWT_SIGNING_PRIVATE_KEY_FILE"`
+	MaxTokenLength              int
+	DefaultApplications         []string
+	EnableAdditionalGrantScopes bool
```

Comment:
Do we really need this configuration item?

Reply:
we don't need the configuration item.

it was introduced with the intention of giving us some time to test it with opt-in enthusiasts and also for those who wanted to try it with their existing tokens. if it didn’t work for them, they could easily revert to the previous setup.

my attempts to gauge how much personal access tokens and oauth2 are being used (by asking in chat channels and searching issues for both gitea and forgejo) led me to the conclusion of "not that much"... though i realize i might be way off here :)

i'll commit the changes with the configuration item removed. if you decide to revert, it will be easy to do so.